src/panels/lovelace/editor/config-elements/hui-map-card-editor.ts

```diff
@@ -155,7 +166,8 @@ export class HuiMapCardEditor extends LitElement implements LovelaceCardEditor {
         }
         this._config = {
           ...this._config,
-          [target.configValue!]: value,
+          [target.configValue]:
+            target.checked !== undefined ? target.checked : target.value,
```

Comment:
you are now using `target.value` instead of `value`

Reply:
Good catch, copy paste error.